Source/WebCore/mathml/MathMLPresentationElement.cpp

```diff
@@ -323,41 +323,41 @@ const MathMLElement::Length& MathMLPresentationElement::cachedMathMLLength(const
 MathMLElement::MathVariant MathMLPresentationElement::parseMathVariantAttribute(const AtomString& attributeValue)
 {
     // The mathvariant attribute values is case-sensitive.
-    if (attributeValue == "normal")
+    if (attributeValue == "normal"_s)
```

Comment:
This really needs to be `SortedArrayMap`; exactly what it’s designed for. Or something that takes advantage of the fact that we have an atom string.

Reply:
Will update.